kyaml/kio/byteio_reader.go

```diff
@@ -47,12 +48,16 @@ type ByteReadWriter struct {
 	NoWrap             bool
 	WrappingAPIVersion string
 	WrappingKind       string
+
+	// RetainSeqIndent if true retains the sequence indentation of
```

Comment:
comment isn't complete. Actually, at the reader level, we can not preserve the seqIndent, we can only record the annotation, so may be we can call it, `AddSeqIndentAnnotation` ?

Also can we move it closer to fields where we have annotation related knobs ?

Reply:
Yes. Updated the name.